extlinks/aggregates/factories.py

```diff
@@ -29,7 +29,7 @@ class Meta:
 
     organisation = factory.SubFactory(OrganisationFactory)
     collection = factory.SubFactory(CollectionFactory)
-    username = factory.Faker("name")
+    username = factory.Sequence(lambda n: 'user%d' % n)
```

Comment:
this doesn't really belong in this pr, but we had a ci failure since our factory didn't respect the new field constraint